test/integration/events/events_test.go

```diff
@@ -38,11 +38,11 @@ import (
 )
 
 func TestEventCompatibility(t *testing.T) {
-	_, ctx := ktesting.NewTestContext(t)
-	ctx, cancel := context.WithCancel(ctx)
-	defer cancel()
-	stopCh := ctx.Done()
+	tCtx := ktesting.Init(t)
+	stopCh := tCtx.Done()
```

Comment:
If others also find `ktesting.TContext` useful (@aojea ?), then I'll rework this PR to use it instead of adding lots of `context.WithCancel` calls.